RecoTracker/LSTCore/README.md

```diff
@@ -1,42 +0,0 @@
-# LSTCore proof of concept
```

Comment:
a readme may be still useful for self-documentation. The content removed here is probably good to go, but we need to eventually add something up to date.

The standalone/README is outdated as well; another to-do "soon"

Reply:
For the general README, let's discuss the content tomorrow at the meeting.

For the standalone one, I can draft an update when I port my SegmentLinking/TrackLooper#408 PR in the cmssw repo and run everything, and Andres can finalize with the details of the CI when these are settled.